proteuslib/flowsheets/RO_with_energy_recovery/monte_carlo_sampling_RO_ERD.py

```diff
@@ -55,3 +55,4 @@ def run_parameter_sweep(results_file, seed=None):
 
 if __name__ == '__main__':
     run_parameter_sweep('output/monte_carlo_results.csv')
+    # run_parameter_sweep('output/monte_carlo_results.csv', seed=1)  # for running testing case
```

Comment:
I put it there because to run the case for testing you need seed=1, it took me a few minutes to figure it out so it'd help anyone who changes the flowsheet and needs to update the monte carlo testing.

Reply:
I would suggest a meaningful comment then, and not just a commented line of code. Something like:

```
# For testing cases, a seed needs to be provided as an additional argument, e.g. seed=1
```

Otherwise this just looks like a left over piece of code that will be ignored